src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -301,6 +303,58 @@ impl Renderer for HtmlHandlebars {
     }
 }
 
+fn make_chapter(
+    item: &Chapter,
+    section: Option<String>,
+    previous: Option<&mut BTreeMap<String, String>>)
+    -> Result<BTreeMap<String, String>> {
+    let mut chapter = BTreeMap::new();
+
+    if let Some(section) = section {
+        chapter.insert("section".to_owned(), section);
+    }
+
+    chapter.insert("name".to_owned(), item.name.to_owned());
+
+    let path = item.path.to_str().ok_or_else(|| {
+        io::Error::new(io::ErrorKind::Other, "Could not convert path to str")
```

Comment:
I believe we're using `error-chain` here, so you should be able to do `chain_err(|| "Could not convert path to str")` instead of using an `io::Error`.